src/MainPage/ChannelsList.module.scss

```diff
@@ -1,111 +1,103 @@
-.channelsListContainer  {
-    font-size: 16px;
-    color: rgb(188,171,188);
-     .channelsListH2 {
-        color: white;
-        font-size: 20px;
-        padding: 8px 0;   
-        border-bottom: 0.5px solid rgba(255,255,255,0.1);
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        margin: 0;
-        height: 40px;
+.channelsListContainer {
+  font-size: 16px;
+  color: rgb(188, 171, 188);
+  .channelsListH2 {
+    color: white;
+    font-size: 20px;
+    padding: 8px 0;
+    border-bottom: 0.5px solid rgba(255, 255, 255, 0.1);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    height: 40px;
+  }
+  .channelsHeaderContainer {
+    display: flex;
+    align-items: center;
+    padding: 15px 20px 10px 5px;
+    padding-left: 15px;
+    cursor: pointer;
+
+    .dropArrowIcon {
+      padding: 3px;
+      margin-right: 2px;
+      font-size: 18px;
+      &:hover {
+        background-color: rgba(255, 255, 255, 0.2);
+        border-radius: 3px;
+      }
+    }
+    .channelsListH3 {
+      font-weight: 700;
+      margin: 0;
+      padding-left: 2px;
     }
-    .channelsHeaderContainer {
-        display: flex;
-        align-items: center;
-        padding: 15px 20px 10px 5px;
-        padding-left: 15px;
-        cursor: pointer;
 
-        .dropArrowIcon {
-            padding: 3px;
-            margin-right: 2px;
-            font-size: 18px;
-            &:hover {
-                background-color: rgba(255,255,255,0.2);
-                border-radius: 3px;
-            }
-        }
-        .channelsListH3 {
-            font-weight: 700;
-            margin:0;
-            padding-left: 2px;  
-        }
-        
-        .channelsVisible {
-            transform: rotate(90deg);
-            transition: 0.3s;
-        }
-        .plusIcon {
```

Comment:
Musisz pamiętać o jednej konwencji, albo przed nazwą klasy w css robisz enter, albo nie, teraz jest przemieszane.

Proponuje przed każdym nowym selektorem (.class, button itp.) zachowywać jedną linię odstępu we wszystkich plikach scss

Reply:
Czy po pseudo klasie np &:hover dla danego elementu też dać spacje?